signer/src/proto/convert.rs

```diff
@@ -1593,14 +1593,21 @@ impl From<DkgPublicShares> for proto::SignerDkgPublicShares {
             dkg_id: value.dkg_id,
             signer_id: value.signer_id,
             commitments: value.comms.into_iter().map(|v| v.into()).collect(),
+            kex_public_key: Some(proto::Point::from(value.kex_public_key)),
         }
     }
 }
 
 impl TryFrom<proto::SignerDkgPublicShares> for DkgPublicShares {
     type Error = Error;
     fn try_from(value: proto::SignerDkgPublicShares) -> Result<Self, Self::Error> {
+        let kex_public_key = match value.kex_public_key {
+            Some(key) => Point::try_from(key)?,
+            // use point at infinity since this value will fail deserialization
```

Comment:
This would make me have even more questions, like "why would we want to fail deserialization", and "what does this have to do with deserialization, we've already deserialized the protobuf by the time that we are here", and things like that. In a few months, I may not be able to remember enough context to answer these questions and it may not be clear that what is here is correct -- or correct anymore given some other changes that have taken place. So the goal is to write enough so that someone who wasn't around for the teams discussions or doesn't recall them, like myself in a few months, could know whether this is correct and non-harmful.

Reply:
Done